app/assets/stylesheets/groups.scss

```diff
@@ -532,4 +532,26 @@
   }
 }
 
+.select2-container .select2-selection--multiple {
```

Comment:
These styles are not specific to groups page, right? In the case move to select2_overrides.scss file